ampd/src/handlers/evm_verify_worker_set.rs

```diff
@@ -0,0 +1,124 @@
+use error_stack::ResultExt;
+use serde::Deserialize;
+
+use axelar_wasm_std::voting::PollID;
+use events_derive::try_from;
+
+use crate::types::{EVMAddress, Hash, TMAddress, U256};
+
+#[derive(Deserialize, Debug)]
+pub struct Operators {
+    pub weights: Vec<(EVMAddress, U256)>,
+    pub threshold: U256,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct WorkerSet {
+    pub tx_id: Hash,
+    pub log_index: u64,
```

Comment:
what is this information here? is that for verifying that a worker set has been updated?

Reply:
This is the information for us to locate the EVM event and do comparison for voting.

Reply:
Maybe call it `WorkerSetConfirmation` or something like that? It's weird to call it just `WorkerSet` with that information